.github/workflows/ecr-build.yml

```
@@ -0,0 +1,47 @@
+concurrency:
+  group: ecr-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  docker-build:
+    uses: ./.github/workflows/reusable-ecr.yml
+    with:
+      environment: dev
+      aws-account-id: ${{ vars.AWS_ACCOUNT_ID }}
+      aws-ecr-repo: ${{ vars.AWS_ACCOUNT_ID }}.dkr.ecr.${{ vars.AWS_REGION }}.amazonaws.com/genai
+      push-to-ecr: ${{ contains(github.event.pull_request.labels.*.name, 'ecr-push') || github.event_name == 'merge_group' }}
```

actionlint (1.7.4)

input "push-to-ecr" is typed as string by reusable workflow "./.github/workflows/reusable-ecr.yml". bool value cannot be assigned

(expression)